test/test_p4est_2d.jl

```
@@ -117,6 +117,28 @@ end
     end
 end
 
+@trixi_testset "elixir_advection_meshview.jl" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_advection_meshview.jl"),
+                        l2=[0.00013773915040249946],
+                        linf=[0.0010140184322192658])
```

If I understand correctly, this elixir should return the same errors as for the elixir examples/p4est_2d_dgsem/elixir_advection_basic.jl. As far as I can tell, the only difference (apart from the `P4estMesh` vs `P4estMeshView` of course) is the `initial_refinement_level`, which is `1` in elixir_advection_basic.jl and `0` in the new elixir elixir_advection_meshview.jl. If it's true that the errors should be the same if we use the same `initial_refinement_level`, I would like to test that, i.e. either set `initial_refinement_level = 1` directly in the new elixir or in the test here. Do you agree?

---

Good idea. I just ran the two runs and they result into the same errors:
(l2 = [0.0001377391504025094], linf = [0.0010140184322170454])

---

That's nice, but could we also test that please?

---

Just added it.

---

Thanks!